rpmfusion-free-rawhide.repo

```diff
@@ -3,7 +3,8 @@ name=RPM Fusion for Fedora Rawhide - Free
 #baseurl=http://download1.rpmfusion.org/free/fedora/development/$basearch/os/
 metalink=https://mirrors.rpmfusion.org/metalink?repo=free-fedora-rawhide&arch=$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
```

Comment:
What do you mean by "your repository" ? It's not mine at all, it's a community driven project. So it's not less "your" responsibility than mine. IMHO, it's a feature that you care to have implemented, so I'm going to assign the work for YOU. 
Here are you tasks: 
- Implement freeipa ansible role taken from fedora infra on your own test VM
- Implement fedmsg ansible role taken from fedora infra from on your own test VM
Then we will see what's next.